snapcraft.yaml

```diff
@@ -508,6 +508,32 @@ parts:
       - lib/*/liblvm*
       - lib/*/libreadline.so*
 
+  minio:
+    build-attributes: [core22-step-dependencies]
+    source: https://github.com/minio/minio
+    source-type: git
+    plugin: nil
+    build-snaps:
+      - go
+    override-pull: |
+      snapcraftctl pull
+    override-build: |
+      set -ex
+
+      # Setup build environment
+      export GOPATH=$(realpath ./.go)
+      export CGO_CFLAGS="-I${SNAPCRAFT_STAGE}/include/ -I${SNAPCRAFT_STAGE}/usr/local/include/"
+      export CGO_LDFLAGS="-L${SNAPCRAFT_STAGE}/lib/ -L${SNAPCRAFT_STAGE}/usr/local/lib/"
```

Comment:
Those shouldn't be needed as minio doesn't need linking against dqlite, liblxc or similar.

Reply:
Ok thanks, im going to pin the version tag too.

Reply:
Fixed.